README.md

```diff
@@ -4,7 +4,7 @@
 [![JetBrains incubator project](https://jb.gg/badges/incubator.svg)](https://github.com/JetBrains#jetbrains-on-github)
 [![Kotlin](https://img.shields.io/badge/kotlin-2.1-blue.svg?logo=kotlin)](http://kotlinlang.org)
 [![CI status](https://img.shields.io/github/checks-status/JetBrains/koog/main)](https://github.com/JetBrains/koog/actions?query=branch%3Amain)
-[![GitHub license](https://img.shields.io/github/license/JetBrains/koog)](LICENSE.txt)
```

Comment:
btw I get a correct image with the old link, isn't it a browser caching issue?

Reply:
Yey, it's finally updated! It was cached on the server side, I was testing it using `curl`.